package/utils/px5g-wolfssl/px5g-wolfssl.c

```diff
@@ -0,0 +1,354 @@
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdio.h>
+#define _GNU_SOURCE
+#include <string.h>
+#include <wolfssl/options.h>
+#include <wolfssl/wolfcrypt/asn.h>
+#include <wolfssl/wolfcrypt/asn_public.h>
+#include <wolfssl/wolfcrypt/ecc.h>
+#include <wolfssl/wolfcrypt/error-crypt.h>
+#include <wolfssl/wolfcrypt/rsa.h>
+#include <wolfssl/wolfcrypt/settings.h>
+
+#define HEAP_HINT NULL
+#define FOURK_SZ 4096
+#define WOLFSSL_MIN_RSA_BITS 2048
+
+enum {
+  EC_KEY_TYPE = 0,
+  RSA_KEY_TYPE = 1,
+};
+
+int write_file(byte *buf, int bufSz, char *path) {
+  int ret;
+  FILE *file;
+  if (path) {
+    file = fopen(path, "wb");
+    if (file == NULL) {
+      perror("Error opening file");
+      exit(1);
+    }
+  } else {
+    file = stdout;
+  }
+  ret = (int)fwrite(buf, 1, bufSz, file);
+  if (path) {
+    fclose(file);
+  }
+  if (ret > 0) {
+    /* ret > 0 indicates a successful file write, set to zero for return */
+    ret = 0;
+  }
+  return ret;
+}
+
+int write_key(ecc_key *ecKey, RsaKey *rsaKey, int type, int keySz, char *fName,
+              bool write_pem) {
+  int ret;
+  byte der[FOURK_SZ], pem[FOURK_SZ];
+  int derSz, pemSz;
+  XMEMSET(der, 0, FOURK_SZ);
```

Comment:
Just initialize with {}. No need to call some special macro.

Reply:
Done & works, thanks!